src/Entity/UserConsentInterface.php

```diff
@@ -13,6 +13,21 @@
  */
 interface UserConsentInterface extends ContentEntityInterface, EntityChangedInterface, EntityOwnerInterface {
 
+  /**
+   * The user only visits agreement page.
+   */
+  const STATE_UNDECIED = 0;
+
+  /**
+   * The user submitted agreement form but do not set checkbox.
+   */
+  const STATE_NOT_AGREE = 1;
+
+  /**
+   * The user has given consent on the current version of data policy.
+   */
+  const STATE_AGRRE = 2;
```

Comment:
@OH17L this is a typo. Should be `STATE_AGREE`

Reply:
@bramtenhove, you are right. I will fix it.